ggjxx/src/game/game.py

```diff
@@ -192,4 +192,16 @@ def trigger_event(self, event):
     def subscribe(self, listener):
         if not hasattr(listener, "trigger"):
             return
-        self.listeners.append(listener)
\ No newline at end of file
+        self.listeners.append(listener)
+
+    def update(self):
+        self.update_fow()
+
+    def update_fow(self):
+        for robot in self.map.robots:
+            x = robot.getX()
+            y = robot.getY()
+
+            self.map.get_cell(x, y).disableFow()
+            for cell in self.getAdjacentCells(x, y):
+                cell.disableFow()
```

Comment:
Te iba a decir que escribiste mal "fog" jaja

Reply:
No me habia decidido entre **FOG** of war, o **F**og **O**f **W**ar